templates/android/library/src/main/java/io/appwrite/Client.kt.twig

```diff
@@ -151,14 +152,31 @@ class Client @JvmOverloads constructor(
     }
 
     /**
-     * Set endpoint
+     * Set endpoint and realtime endpoint.
      * 
      * @param endpoint
      *
      * @return this     
      */
     fun setEndpoint(endPoint: String): Client {
         this.endPoint = endPoint
+
+        if (this.endPointRealtime == null && endPoint.startsWith("http")) {
```

Comment:
There's also a case where the endpoint starts with `https` where we need to use `wss`

Reply:
In that case `startsWith("http")` is true and this block only replaces `http` with `ws`, so `https` will become `wss`

Reply:
Ah right! Perfect :+1: